ldi-orchestrator/ldio-connectors/ldio-http-in-poller/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/HttpInputPoller.java

```diff
@@ -28,13 +31,14 @@ public class HttpInputPoller extends LdiInput {
 	private static final Logger LOGGER = LoggerFactory.getLogger(HttpInputPoller.class);
 	private static final String CONTENT_TYPE = "Content-Type";
 
-	public HttpInputPoller(ComponentExecutor executor, LdiAdapter adapter, List<String> endpoints,
-			boolean continueOnFail, RequestExecutor requestExecutor) {
-		super(executor, adapter);
+	public HttpInputPoller(String componentName, String pipelineName, ComponentExecutor executor, LdiAdapter adapter, List<String> endpoints,
+	                       boolean continueOnFail, RequestExecutor requestExecutor) {
+		super(componentName, pipelineName, executor, adapter);
 		this.requestExecutor = requestExecutor;
 		this.requests = endpoints.stream().map(endpoint -> new GetRequest(endpoint, RequestHeaders.empty())).toList();
 		this.continueOnFail = continueOnFail;
 		this.scheduler = Executors.newSingleThreadScheduledExecutor();
+		Metrics.counter(LDIO_DATA_IN, PIPELINE, pipelineName, LDIO_COMPONENT_NAME, componentName).increment(0);
```

Comment:
What is the point of this? Why do we increment with 0?

Reply:
This is pure for Prometheus. If not, the counter will not show up. But might need to double check on this